Resources/Prototypes/_Scp/Maps/scp_complex_standard.yml

```
@@ -36,7 +36,7 @@
             SeniorLogisticsOfficer: [ 1, 1 ]
             # Отдел внутренней безопасности
             #MentalControlOfficer: [ 1, 1 ]
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)